draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1901,9 +1904,66 @@ and feedback.
+
+Finally IANA should track UUID Variants, UUID Subtypes, and Special Case "Reserved UUIDs" such as Min, Max, and Namespace IDs as specified in {{iana1}}, {{iana2}}, {{iana3}}.
+
+The policy for these three registries is "Specification Required", per {{RFC8126}}, Section 4.6.
+
+When evaluating requests, the designated expert(s) should consider community feedback, how well-defined is the reference specification, and this specification's requirements.
+Vendor-specific, application-specific, and deployment-specific values are unable to be registered.
+Specification documents should be published in a stable, freely available manner (ideally located with a URL) but need not be standards.
+The designated experts will either approve or deny the registration request, and communicate their decision to IANA. Denials should include an explanation and, if applicable, suggestions as to how to make the request successful.
+
+## IANA UUID Variants Registry and Registration {#iana1}
+This specification defines the "UUID Variants" registry for common, widely used UUID standards.
+
+{{ianaVariants}} should be used as-is for this registry with "This document" replaced as required.
+
+| Name           | Bits | Variant | Reference                  |
+| Apollo NCS     | 0xxx | 1-7     | {{RFC4122}}, This document |
+| OSF DCE / IETF | 10xx | 8-9,A-B | {{RFC4122}}, This document |
+| Microsoft      | 110x | C-D     | {{RFC4122}}, This document |
+| Unused         | 111x | E-F     | {{RFC4122}}, This document |
+{: #ianaVariants title='IANA UUID Variants'}
+
+## IANA UUID Subtype Registry and Registration {#iana2}
+This specification defines the "UUID Subtype" registry for common, widely used UUID standards.
+
+{{ianaSubtypes}} should be used as-is for this registry with "This document" replaced as required.
+
+| Name                           | ID | Hex | Subtype | Variant        | Reference                  |
+| Unused                         |  0 | 0x0 | Version | OSF DCE / IETF | This document              |
+| Gregorian Time-based           |  1 | 0x1 | Version | OSF DCE / IETF | {{RFC4122}}, This document |
+| DCE Security                   |  2 | 0x2 | Version | OSF DCE / IETF | {{C309}}, {{C311}}         |
+| MD5 Name-based                 |  3 | 0x3 | Version | OSF DCE / IETF | {{RFC4122}}, This document |
+| Random                         |  4 | 0x4 | Version | OSF DCE / IETF | {{RFC4122}}, This document |
+| SHA-1 Name-based               |  5 | 0x5 | Version | OSF DCE / IETF | {{RFC4122}}, This document |
+| Reordered Gregorian Time-based |  6 | 0x6 | Version | OSF DCE / IETF | This document              |
+| Unix Time-based                |  7 | 0x7 | Version | OSF DCE / IETF | This document              |
+| Custom                         |  8 | 0x8 | Version | OSF DCE / IETF | This document              |
+| Unspecified                    |  9 | 0x9 | Version | OSF DCE / IETF | This document              |
```

Comment:
I was unable to find any info on if this was good or bad. I planned on bringing it up in the interim.

Reply:
In case of DNS, not all 65536 possibilities for record types are `Unspecified`, so seems bad practice to me to list them.

Reply:
Should I drop v0? Seems folks forget that's technically a possible version although unused/unspecified.

Reply:
I know it is possible and I once came across someone who implemented a v0 version. In my opinion, I would set that one on `Reserved` instead of `Unspecified`.